satellite/console/consoleweb/server.go

```diff
@@ -167,7 +187,7 @@ func (s *Server) bucketUsageReportHandler(w http.ResponseWriter, req *http.Reque
 		return
 	}
 
-	report, err := template.ParseFiles(path.Join(s.config.StaticDir, "static", "reports", "UsageReport.html"))
+	report, err := template.New("UsageReport.html").Parse(s.getFromArchive("static/reports/UsageReport.html"))
```

Comment:
i hope this works. what if the template depends on another template? hmm. let's try it for now, but this may be a source of template bugs. we might need to figure out how to get something like ParseFiles working